docs/MO_DG/prepare_model/customize_model_optimizer/Customize_Model_Optimizer.md

```diff
@@ -1,222 +1,228 @@
-# Model Optimizer Extensibility {#openvino_docs_MO_DG_prepare_model_customize_model_optimizer_Customize_Model_Optimizer}
+# [LEGACY] Model Optimizer Extensibility {#openvino_docs_MO_DG_prepare_model_customize_model_optimizer_Customize_Model_Optimizer}
```

Comment:
Still don't see a good explanation why this chapter is considered to be legacy. Please add something like described here: https://docs.openvino.ai/latest/openvino_docs_Extensibility_UG_Intro.html#mapping-from-framework-operation about why two extensibility exists. Focus on explaining that the code described here will be deprecated soon and kept only for backward compatibility and old users. I'm not sure about precise words, but it should clearly stops new users to continue reading this document.